### PR TITLE
Dockerizing it.

### PR DESCRIPTION
**Pull request by Fumenoid**
Dockerising the bot
Added requirements.txt + some minor changes
_Added my name in cool people too_

### CODE REVIEW

File: nullctf.py [hunk 1]
```diff
@@ -91,6 +93,11 @@ async def amicool(ctx):
         await ctx.send('Psst, kid.  Want to be cool?  Find an issue and report it or request a feature!')
 
 if __name__ == '__main__':
+    sys.path.insert(1, os.getcwd() + '/cogs/')
     for extension in extensions:
-        bot.load_extension('cogs.' + extension)
+        try:
+            bot.load_extension(extension)
+        except Exception as e:
+            logger.critical(f'Failed to load cogs: {e}')
```
Comment (NullPxl) [inline nullctf.py:101]: Getting an "undefined" error here.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+discord.py
+pymongo
```
Comment (NullPxl) [inline requirements.txt:2]: dnspython should also be included

**Comment by NullPxl** [comment]
Thanks! I added some comments with a couple issues I had but other than that it's good to go 👍

File: nullctf.py (hunk 1 above)
Comment (Fumenoid) [reply nullctf.py:101]: wait, I need to correct it

File: requirements.txt (hunk 2 above)
Comment (Fumenoid) [reply requirements.txt:2]: Alright, on it !

**Comment by Fumenoid** [comment]
Did some corrections, logger wasn't required.
Added dnspython in requirements.txt and twerked help menu a bit for readability so check that too.

**Comment by NullPxl** [comment]
Great! Thanks again.

**Pull request merged**